src/plugins.c

```diff
@@ -1344,6 +1347,10 @@ void plugins_init(void)
 	g_signal_connect(geany_object, "save-settings", G_CALLBACK(update_active_plugins_pref), NULL);
 	stash_group_add_string_vector(group, &active_plugins_pref, "active_plugins", NULL);
 
+	builtin_proxy_glob = g_strconcat("*.", G_MODULE_SUFFIX, NULL);
+	builtin_so_proxy.glob = g_pattern_spec_new(builtin_proxy_glob);
```

Comment:
This would probably be better like `g_pattern_spec_new("*." G_MODULE_SUFFIX);` rather than building a dynamic string and then freeing it right away.